metacatalog/ext/standards_export/extension.py

```diff
@@ -78,6 +81,17 @@ def wrapper_api(session: Session, id_or_uuid: Union[int, str], config_dict: dict
         api.catalog.create_iso19115_xml = wrapper_api
 
 
+    @classmethod
+    def init_cli(cls, subparsers, defaults):
+        myparser = subparsers.add_parser('standards-export', parents=[defaults], help="Export metadata in standard format as .xml files.")
+        myparser.add_argument('--format', choices=['iso19115'], type=str, required=True, help="Metadata standard format.")
```

Comment:
There is a way to set a default of the flag is not set. I would change this to not-required but set iso19115 as a default if none is set.